augur/application/db/util.py

```diff
@@ -1,20 +1,20 @@
-from sqlalchemy.exc import OperationalError
 import time
+from typing import Callable, Any, List, Dict, Optional, Union
+from sqlalchemy.exc import OperationalError
 
 
-
-def catch_operational_error(func):
-
+def catch_operational_error(func: Callable[[], Any]) -> Any:
+    """
+    Executes a function and retries up to 4 times if OperationalError occurs.
+    Implements exponential backoff starting with 240 seconds.
+    """
     attempts = 0
-    error = None
+    error: Optional[str] = None
     timeout = 240
 
     while attempts < 4:
-
-        # do the sleep here instead of instead of in the exception 
-        # so it doesn't sleep after the last failed time
         if attempts > 0:
-            #Do a 30% exponential backoff
+            # Do a 30% exponential backoff
```

Comment:
this seems like a non-typing change that removed a helpful comment - was this removed on purpose?